themes/startwords/assets/scss/_fonts.scss

```
@@ -121,7 +121,4 @@
     // harmonize sizing with font for Latin characters
     // on desktop, main font should be 18px and Noto Serif TC should be 16px
     size-adjust: 89%;
```

I think we need to keep the font-weight, just adjust it to match the font style where this font will be used.

```suggestion
    size-adjust: 89%;
    font-weight: 400;
```